atlasdb-client/src/main/java/com/palantir/atlasdb/tracing/Tracing.java

```diff
@@ -21,16 +21,26 @@
 import com.google.errorprone.annotations.CompileTimeConstant;
 import com.palantir.atlasdb.keyvalue.api.TableReference;
 import com.palantir.atlasdb.logging.LoggingArgs;
+import com.palantir.logsafe.SafeArg;
 import com.palantir.tracing.CloseableTracer;
 import com.palantir.tracing.TagTranslator;
 import java.util.Collection;
 import java.util.Map;
 import java.util.function.BiConsumer;
 import java.util.function.Consumer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public interface Tracing {
+    Logger log = LoggerFactory.getLogger(Tracing.class);
 
     static CloseableTracer startLocalTrace(@CompileTimeConstant String operation, Consumer<TagConsumer> tagTranslator) {
+        CloseableTracer tracer = CloseableTracer.startSpan(operation, FunctionalTagTranslator.INSTANCE, tagTranslator);
+        log.info(
+                "startLocalTrace",
+                SafeArg.of("operation", operation),
+                SafeArg.of("tracer class", tracer.getClass()),
+                SafeArg.of("trace", tracer.toString()));
```

Comment:
tracer.toString() doesn't do anything interesting. I think a better thing is Tracer#maybeGetTraceMetadata or something from that class that will get you the traceId so you can log it. 

Actually, the logline itself will contain the traceId, so probably just remove tracer.toString()

Reply:
Actually: possibly the actual important information is to log the whole TraceMetadata: just traceId possibly isn't the whole picture, if the parentSpanId is incorrectly parented. So yeah, I think we want to log the TraceMetadata obtained from Tracer#maybeGetTraceMetadata in all cases.